pkg/ocicni/ocicni.go

```diff
@@ -167,12 +183,7 @@ func InitCNI(pluginDir string, cniDirs ...string) (CNIPlugin, error) {
 		return nil, err
 	}
 
-	if err := plugin.syncNetworkConfig(); err != nil {
-		// We do not have a valid default network, so start the
-		// monitoring thread.  Network setup/teardown requests
-		// will fail until we have a valid default network.
-		go plugin.monitorNetDir()
-	}
+	go plugin.monitorNetDir()
```

Comment:
I believe it is a bug initially. `monitorNetDir` doesn't return after `syncNetworkConfig`, so I believe we want it to keep syncing network config. But we don't always start the loop.

Actually in cni in kubernetes, we do keep syncing network config https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/cni/cni.go#L200.

Given so, I think we should accept this change.